packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.constants.js

```diff
@@ -1,5 +1,11 @@
 export const SOFTPHONE_TYPE = 'softphone';
+export const MOBILE_OS = {
+  windows: 'windows',
+  android: 'android',
+  ios: 'ios',
```

Comment:
To be consistent with other contants, shouldn't those be in upper case ?

Reply:
I am not sure, I can find both 
![image](https://github.com/ovh/manager/assets/8767496/e202304b-00f8-4da3-a61b-26f067c78ff2)